test/IRGen/enum_empty_payloads.sil

```diff
@@ -1,19 +1,21 @@
 // RUN: %target-swift-frontend -assume-parsing-unqualified-ownership-sil -emit-ir -verify %s
 sil_stage canonical
 
+typealias Void = ()
+
 struct Empty<T> {}
 
 enum SinglePayload<T> {
   case A(T)
-  case B()
+  case B(Void)
```

Comment:
I suspect this was trying to test `case B` rather than `case B(Void)`. @gottesmm?

Reply:
We should make sure both have the same effect in IRGen, since tuples should be exploded recursively. In the past IRGen has had bugs with empty types so we should make sure its tested.